Refresh.GameServer/Endpoints/Game/CommentEndpoints.cs

```diff
@@ -82,6 +88,12 @@ public Response PostLevelComment(RequestContext context, GameDatabaseContext dat
         GameLevel? level = database.GetLevelByIdAndType(slotType, id);
         if (level == null) return NotFound;
 
+        
+        int count = database.GetNotificationCountByUser(level.Publisher);
+        if (!level.Publisher.Equals(user) && count < 10) {
+            database.AddNotification("New level comment", $"{user.Username} left a comment on your level: '{level.Title}!'", level.Publisher);
```

Comment:
```suggestion
            database.AddNotification("New comment", $"{user.Username} left a comment on your level: '{level.Title}!'", level.Publisher);
```

Reply:
HOLD ON, BY NOTIFICATIONS I MEANT RECENT ACTIVITY NOTIFICATIONS

Reply:
> HOLD ON, BY NOTIFICATIONS I MEANT RECENT ACTIVITY NOTIFICATIONS

doesn't matter for this PR, we are still going to accept it since this is a nice feature (not everyone wants to check ingame/on recent activity to see this), but you should clarify in your issues that "notifications" means "recent activity events", since those are distinctly different things.

Reply:
yeah, sorry... i just didn't know what to call them really

i hope it's fine

Reply:
You can make a new issue but please stop commenting on the PR.